syntax = "proto2";

package Arcusical.LocalMusicStore;

import "CachedSong.proto";

message CachedSongList
{
	repeated CachedSong songs = 1;
}